src/widgets/attachments/image-preview.component.tsx

```diff
@@ -38,24 +39,33 @@ export default function ImagePreview(props: ImagePreviewProps) {
         }
         alt={t("webcamPreview", "Webcam preview")}
       />
-      <input
-        type="text"
-        placeholder={t(
-          "attachmentCaptionInstruction",
-          "Enter a caption for the image"
-        )}
-        onChange={updateCaption}
-      />
-      <UserHasAccess privilege="Create Attachment">
-        <button onClick={saveImage}>{t("save", "Save")} </button>
-      </UserHasAccess>
-      <button onClick={cancelCapture}>{t("cancel", "Cancel")} </button>
+      {props.collectCaption && (
+        <input
+          type="text"
+          placeholder={t(
+            "attachmentCaptionInstruction",
+            "Enter a caption for the image"
+          )}
+          onChange={updateCaption}
+        />
+      )}
+      <ButtonSet style={{ width: "50%" }}>
+        <UserHasAccess privilege="Create Attachment">
```

Comment:
If the only thing we can do is to cancel - why have something at all here?

Shouldn't the whole button set / form be hidden if no privileges are there?

Reply:
Resolved.